crypto/rand/rand_lib.c

```diff
@@ -402,7 +402,7 @@ int RAND_poll(void)
     } else {
         /* fill random pool and seed the current legacy RNG */
         pool = rand_pool_new(RAND_DRBG_STRENGTH,
-                             RAND_DRBG_STRENGTH / 8,
+                             (RAND_DRBG_STRENGTH + 7) / 8,
```

Comment:
This is the most cosmetical change of all, because I doubt that it will ever happen that the security strength is not a multiple of eight. ;-)

Reply:
I kind of agree but better to be safe.

A variable length digest used underneath a hash DRBG?
I might try a SHAKE based DRBG when I get some time :)